packages/instant/src/containers/latest_error_overlay.tsx

```diff
@@ -0,0 +1,25 @@
+import { connect } from 'react-redux';
+import { Dispatch } from 'redux';
+
+import { OverlayContainer, OverlayContainerProps } from '../components/ui/overlay_container';
+
+import { Action } from '../redux/actions';
+import { State } from '../redux/reducer';
+import { ScreenWidths } from '../style/media';
+import { DisplayStatus } from '../types';
+import { errorFlasher } from '../util/error_flasher';
+
+type ConnectedState = Pick<OverlayContainerProps, 'showOverlay' | 'showMaxWidth'>;
```

Comment:
Oh I like this.

Reply:
Yea, I think we could benefit from adopting this pattern for all of our containers.  Would help reduce work when interfaces change.